docs/merged_objects.md

```diff
@@ -224,8 +224,104 @@ Some merged objects may have some additional sample metadata columns specific to
 Examples of this include treatment or outcome.
 
 
+### SingleCellExperiment dimensionality reduction results
 
+The `reducedDim` slot of the merged object will be contain both principal component analysis (`PCA`) and `UMAP` results.
 
+PCA results were calculated using [`batchelor::multiBatchPCA`](https://rdrr.io/bioc/batchelor/man/multiBatchPCA.html), specifying libraries as batches to ensure that each library in the merged object was equally weighted, and specifying a list of highly variable genes.
+The highly variable genes were selected with `scran::modelGeneVar`, with libraries specified as blocks, and `scran::getTopHVGs`, and they are stored in the `SingleCellExperiment` object in `metadata(merged_sce)$merged_highly_variable_genes`.
+The following command can be used to access the PCA results:
+
+```r
+reducedDim(merged_sce, "PCA")
+```
+
+UMAP results were calculated using `scater::runUMAP()`, with the PCA results as input rather than the full gene expression matrix.
+The following command can be used to access the UMAP results:
+
+```r
+reducedDim(merged_sce, "UMAP")
+```
+
+### Additional SingleCellExperiment components for CITE-seq libraries (with ADT tags)
+
+ADT data from CITE-seq experiments, when present, is included within the merged `SingleCellExperiment` object as an "Alternative Experiment" named `"adt"`, which can be accessed with the following command:
+
+```r
+altExp(merged_sce, "adt") # adt experiment
+```
+
+Within this, the main expression matrix is again found in the `counts` assay and the normalized expression matrix is found in the `logcounts` assay.
+For each assay, each column corresponds to a cell or droplet (in the same order as the parent `SingleCellExperiment`) and each row corresponds to an antibody derived tag (ADT).
+In some cases, only some libraries in the merged object will have associated CITE-seq data.
+Libraries which do not have CITE-seq expression data are included in the assay matrices with all `NA` count values.
+Column names are again cell barcode sequences prefixed with the originating library id, e.g. `SCPCL000000-{barcode}`, and row names are the antibody targets for each ADT.
```

Comment:
```suggestion
For each assay, each column corresponds to a cell or droplet (in the same order as the parent `SingleCellExperiment`) and each row corresponds to an antibody derived tag (ADT).
Column names are again cell barcode sequences prefixed with the originating library id, e.g. `SCPCL000000-{barcode}`, and row names are the antibody targets for each ADT.
In some cases, only some libraries in the merged object will have associated CITE-seq data.
Libraries which do not have CITE-seq expression data are included in the assay matrices with all `NA` count values.
```
I think I would just reorder slightly so you talk about the column/row names right after you mention what their contents are.